src/cmd/linuxkit/run_hyperkit.go

```diff
@@ -206,12 +207,15 @@ func runHyperKit(args []string) {
 			id = strconv.Itoa(i)
 		}
 		if d.Size != 0 && d.Path == "" {
-			d.Path = filepath.Join(*state, "disk"+id+".img")
+			d.Path = filepath.Join(*state, "disk"+id+".raw")
```

Comment:
There seems to be some tests which use `.img` still. At least  `test/cases/040_packages/005_extend/000_ext4/test.sh`.

Maybe hyperkit should accept (without the warning) `.img` as a raw suffix in `GetDiskFormat`? I can sort of see why it might be nice to avoid that though since `.img` _is_ a bit ambiguous in terms of what it might contain.

Reply:
Did you mean to keep this now that you've allowed `.img`?

Reply:
yes, `.raw` is just that tiny bit more explicit